tests/integration/targets/syslog_server_info/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
```

Comment:
We cannot run `syslog_server` and `syslog_server_info` in parallel. Move this file to be a `tests/integration/targets/syslog_server/tasks/01_syslog_server_info.yml`.